### PR TITLE
Code smells resolved

### PR DESCRIPTION
**Pull request by swetatech24**

### CODE REVIEW

File: src/main/java/com/iemr/helpline104/data/comoOutbound/T_104CoMoOutboundCallDetails.java [hunk 1]
```diff
@@ -39,7 +39,7 @@ public class T_104CoMoOutboundCallDetails {
 	@GeneratedValue(strategy = GenerationType.IDENTITY)
 	private Long iD;
 	private Integer uSRMappingID;
-	private String callId;
+	private String czentrixCallID;
```
Comment (Anjana2310) [inline src/main/java/com/iemr/helpline104/data/comoOutbound/T_104CoMoOutboundCallDetails.java:42]: The variable is callId

**Review by Anjana2310 — CHANGES_REQUESTED** [review]
@swetatech24 Please address the comments

**Review by Anjana2310 — APPROVED** [review]
@swetatech24 Looks good to me

File: src/main/java/com/iemr/helpline104/data/beneficiary/BeneficiaryRegistrationData.java [hunk 2]
```diff
@@ -37,27 +37,19 @@
 public class BeneficiaryRegistrationData {
 	private List<M_Title> m_Title;
 	private List<M_Status> m_Status;
-	//private List<I_BeneficiaryEducation> i_BeneficiaryEducation;
+	// private List<I_BeneficiaryEducation> i_BeneficiaryEducation;
```
Comment (harimohanr) [inline src/main/java/com/iemr/helpline104/data/beneficiary/BeneficiaryRegistrationData.java:40]: @swetatech24 Why is this commented line still here? Shouldnt we remove this?

File: src/main/java/com/iemr/helpline104/data/beneficiary/BeneficiaryRegistrationData.java [hunk 3]
```diff
@@ -37,27 +37,19 @@
...
 	private List<States> states;
 	private List<M_Gender> m_genders;
 	private List<M_MaritalStatus> m_maritalStatuses;
 	private List<M_Community> m_communities;
 	private List<M_Language> m_language;
 	private List<Directory> directory;
-	//private List<I_BenRelationshipType> i_beneficiaryType;
+	// private List<I_BenRelationshipType> i_beneficiaryType;
```
Comment (harimohanr) [inline src/main/java/com/iemr/helpline104/data/beneficiary/BeneficiaryRegistrationData.java:47]: @swetatech24 Why is this commented line still here? Shouldnt we remove this?

**Review by harimohanr — APPROVED** [review]
@swetatech24 Looks okay to me

**Comment by swetatech24** [comment]
All required changes done

**Pull request merged**